lib/oxidized/model/xos.rb

```diff
@@ -1,7 +1,7 @@
 class XOS < Oxidized::Model
   # Extreme Networks XOS
 
-  prompt /^*?[\w .-]+# $/
+  prompt /^\*?\s?[-\w.~]+(:\d+)? [#>] $/
```

Comment:
Regular XOS-prompts usually look like this `my-device02.1 # ` where the `.1` is a value that gets appended and incremented in the prompt. The section `(:\d)?` was added because there are older XOS-devices that uses a similar prompt, but the dot is replaced with a colon. For example `my-device02:1 # `